sale_coupon_multi_use/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "website": "https://github.com/OCA/sale-workflow",
     "author": "Camptocamp, Odoo Community Association (OCA)",
     "license": "AGPL-3",
-    "depends": ["sale_coupon"],
+    "depends": ["sale_coupon_multi_currency"],
```

Comment:
You have a problem because with your implementation, we can't install `sale_coupon_multi_use` without `sale_coupon_multi_currency`.
Modules must not have dependencies between them.
We must can have a multi use of coupons without multi currencies and the inverse too.

Don't finish the review on multi_use module as you will redo this part.

Reply:
I was thinking about that, but thought that won't make a difference if these are separated. If you want I can glue em.